### ISSUE TITLE
Unable to subscribe in a fund

### ISSUE THREAD

**Issue by Nicolabhub** [melonproject/ipfs-frontend#210]
I would say that 4 times out of 5 if not 9 out of 10, buying shares in a fund results in 
![image](https://user-images.githubusercontent.com/29412458/35651179-0c4099d6-06df-11e8-9c1f-99957429ca7b.png)

**Comment by jennazenk** [comment]
So in the end your fund's AUM increased ?

**Comment by Nicolabhub** [comment]
yes, crazy number of iterations were needed. It must have come from the
chain overload

**Comment by primius** [comment]
I'm getting the negative gas price error too. When trying to buy shares with MLN. After the waiting period, when clicking on the "Execute my request button".
But i haven my AUM hasn't increased by now. I tried it at least 4 times.
